packages/legend-graph/src/helpers/MappingResolutionHelper.ts

```diff
@@ -32,7 +35,7 @@ import { getClassMappingsByClass } from './MappingHelper';
 export const updateRootSetImplementationOnCreate = (
   setImp: SetImplementation,
 ): void => {
-  const classMappingsWithSimilarTarget = getClassMappingsByClass(
+  const classMappingsWithSimilarTarget = getOwnClassMappingsByClass(
```

Comment:
Can you add comment on these why we use `getOwnClassMappings` instead of `getClass...` please?